planemo/galaxy/activity.py

```diff
@@ -375,7 +375,10 @@ def execute_rerun(
     if rerunnable.rerunnable_type == "history":
         job_ids = [job["id"] for job in user_gi.jobs.get_jobs(history_id=rerunnable.rerunnable_id, state="error")]
     elif rerunnable.rerunnable_type == "invocation":
-        job_ids = [job["id"] for job in user_gi.jobs.get_jobs(invocation_id=rerunnable.rerunnable_id, state="error")]
+        request = user_gi.invocations._get(f"{rerunnable.rerunnable_id}/request")
+        request["use_cached_job"] = True
```

Comment:
Should this be an option for the planemo rerun command?

```suggestion
        request["use_cached_job"] = kdws.get("use_cached_job", True)
```

Reply:
Made it configurable and adjusted the docs, thank you @nsoranzo!